src/components/NewWorkspaceModal.js

```diff
@@ -379,24 +379,17 @@ const NewWorkspaceModal = withDisplayName(
                     h(Fragment, [
                       h(FormLabel, { htmlFor: id }, [
                         h(Checkbox, {
-                          'aria-label': 'enhancedBucketLoggingToggle',
-                          style: { margin: '0.25rem 0.25rem 0.25rem 0' },
+                          // 'aria-label': 'enhancedBucketLoggingToggle',
+                          style: { margin: '0 0.25rem 0.25rem 0' },
                           id,
                           checked: enhancedBucketLogging,
                           disabled: requireEnhancedBucketLogging || groups.length > 0,
                           onChange: () => setEnhancedBucketLogging(!enhancedBucketLogging),
                         }),
                         'Enhanced Bucket Logging',
                         h(InfoBox, { style: { marginLeft: '0.25rem', padding: '0' } }, [
-                          '"Enhanced Bucket Logging" means ...',
-                          h(
-                            Link,
-                            {
-                              href: 'https://youtu.be/E4WlUXrJgy4',
-                              ...Utils.newTabLinkProps,
-                            },
-                            ['Read more about Enhanced Bucket Logging']
-                          ),
+                          'Enabling Enhanced Bucket Logging tells the Terra to log all data access requests to the workspace bucket. ' +
```

Comment:
Do users know the import of this, or should we mention that it's required for some protected data sets?